indigo/templatetags/indigo_tags.py

```diff
@@ -0,0 +1,23 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def simplify_field_title(value):
```

Comment:
I would have preferred the data standard to not have the "Select a value:" strings in it - the data standard may be used in other contexts apart from a form (eg data validation), and there "select" doesn't make sense. (and hence this function wouldn't have been needed) However now the data standard has been released let's go with it.

Reply:
Yeah, sorry I may have misued this field ... I copied the schema from Projects and the form labels `"title": "(Value)",` didn't make much sense in the context of the form input, so I tried to make it clearer to the user.

Reply:
There are plans to add new fields to the schema for policy sector fairly soon, so maybe I can find a better way of doing this then.